### PR TITLE
Wave2 1104 log error cleanup

### PR DESCRIPTION
**Pull request by balrajb**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core.module [hunk 1]
```diff
@@ -18,21 +18,26 @@
...
+      } // is this first item?
     }
+  }
 
-    if (!is_null($found) && $found == 0) return true; // if found isn't null, we found it
-    return false; // we didn't
+  if (!is_null($found) && $found == 0) {
+    return TRUE;
+  } // if found isn't null, we found it
+  return FALSE; // we didn't
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core.module]: Should be indented like line 38.

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core.module [hunk 2]
```diff
@@ -18,21 +18,26 @@
...
+      if ($item['value'] == $fc_item_id) {
+        $found = $key;
+      } // is this first item?
     }
+  }
 
-    if (!is_null($found) && $found == 0) return true; // if found isn't null, we found it
-    return false; // we didn't
+  if (!is_null($found) && $found == 0) {
+    return TRUE;
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core.module]: add comment // we found it

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core.module [hunk 3]
```diff
@@ -179,31 +187,32 @@ function ucgy_core_tokens($type, $tokens, array $data = array(), array $options
...
-                /*                if ($item && $item->type == 'fieldable_panels_pane') {
-                                    $fpp = fieldable_panels_panes_load_from_subtype($item->subtype);
-
-                                    if (is_object($fpp)) {
-                                        break;
-                                    }
-                                }*/
-            }
-        }
+        /*                if ($item && $item->type == 'fieldable_panels_pane') {
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core.module]: This code is commented out. Should be removed.

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core.module [hunk 4]
```diff
@@ -613,7 +613,7 @@ function ucgy_core_panels_pre_render($panels_display, $renderer)
 
     // We need to load the FPP. But first, let's make sure it *is* an FPP
     // All FPPs have a subtype that starts "uuid:"
-    if (strpos($pane->subtype,'uuid:') !== false) {
+    if (strpos($pane->subtype,'uuid:') !== false || strpos($pane->subtype,'current:' ) !== FALSE) {
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core.module]: Both FALSE statements should follow the same format (e.g. FALSE vs false)

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core.module [hunk 5]
```diff
@@ -741,7 +741,7 @@ function _ucgy_core_get_fpp_from_subtype($subtype = null) {
 
   // We need to load the FPP. But first, let's make sure it *is* an FPP
   // All FPPs have a subtype that starts "uuid:"
-  if (strpos($subtype,'uuid:') !== false) {
+  if (strpos($subtype,'uuid:') !== false || strpos($subtype,'current:' ) !== FALSE ) {
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core.module]: Same issue here, follow one format or the other.

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core.module (hunk 2 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core.module]: Fixed.

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core.module (hunk 1 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core.module]: Fixed.

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core.module (hunk 3 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core.module]: I wasn't sure about this one. This code seems to be left in on purpose as part of a TODO item. With it commented out, the if statement it resides in is useless. And with that if statement being useless, the entire function should be reviewed.

Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core.module]: Looks like this function isn't being called anywhere. Should I remove it?

**Comment by taeram** [comment]
Closing this and re-applying the changes in a new Pull Request to remove all of the code style changes and more easily review the actual code changes.

**Pull request closed without merging**